transform/mattermost-analytics/models/marts/product/nps/fct_nps_feedback.sql

```diff
@@ -1,4 +1,5 @@
 select {{ dbt_utils.generate_surrogate_key(['nf.server_id', 'nf.user_id', 'nf.feedback', 'nf.event_date']) }} as nps_feedback_id
+       , {{ dbt_utils.generate_surrogate_key(['nf.event_date', 'nf.server_id']) }} AS daily_server_id,
```

Comment:
Order of fields seems to be different in the majority of the models (`server_id` then date column). Seems there's a bug in `fct_nps_score` as well. Let's use the following suggestion here: 

```suggestion
       , {{ dbt_utils.generate_surrogate_key(['nf.server_id', 'nf.event_date']) }} AS daily_server_id,
```